### DISCUSSION TITLE
[OP Stack Deployment] <Where to find my contract addresses after deploying branch op-contracts/v1.3.0?>

### DISCUSSION THREAD

**Discussion by rs-capybaralabs** [ethereum-optimism/developers#412] [Deploying the OP Stack]
### Did you check the documentation?

- [X] Yes

### Did you check for duplicate questions?

- [X] Yes

### Issue Description

Hi community, i'm trying to deploy a new chain with node op-node/v1.7.2 and contract op-contracts/v1.3.0. It used to work for me but in recent cases I can't find all l1 contract addresses after calling `forge script scripts/Deploy.s.sol:Deploy --private-key $GS_ADMIN_PRIVATE_KEY --broadcast --rpc-url $L1_RPC_URL --slow`. There were files generated under /packages/contracts-bedrock/deployments/getting-started previously stating all the address so I can fill them into a l1-deployments file. Where should I find them?

**Comment by rs-capybaralabs** [comment]
Btw, do you have a suggested version combination for opnode/batcher/proposer, op-geth and op-contracts?

**Comment by Chomtana** [comment]
Contract addresses are in `deployments/.../.deploy` file

You should deploy contracts using `op-contracts/v1.3.0` branch and then switch back to the latest released branch ([op-node/v1.7.5](https://github.com/ethereum-optimism/optimism/tree/op-node/v1.7.5)) to run op-node/op-batcher/op-proposer. Same with op-geth, look into op-geth repository.

**Reply by Chomtana** [reply]
Yes, contract and deployment files should be generated from `op-contracts/v1.3.0`

However, newver version has been released but the fault proof instructions are still unknown.

And program should be compiled and run with the latest release.

You can see latest release here
* op-geth: https://github.com/ethereum-optimism/op-geth/releases
* other: https://github.com/ethereum-optimism/optimism/releases (Ignore op-contracts release and scroll down until you found a normal release)
